draft-ietf-core-resource-directory.md

```diff
@@ -613,8 +613,8 @@ The use of multicast discovery in mesh networks is NOT RECOMMENDED.
 
 ### Resource Directory Address Option (RDAO) {#rdao}
 
-The Resource Directory Address Option (RDAO) using IPv6 Neighbor Discovery (ND) carries
-information about the address of the RD. This information is
+The Resource Directory Address Option (RDAO) carries
+information about the address of the RD in RAs (Route Announcements) of IPv6 Neighbor Discovery (ND). This information is
```

Comment:
Route Announcements ➔ Router Advertisement

Reply:
Maybe explicitly mention that this is analogous to the way the RDNSS option in RFC 8106 works.